examples/timestamping/backend/src/transactions.rs

```diff
@@ -46,18 +47,28 @@ impl From<Error> for ExecutionError {
     }
 }
 
-transactions! {
-    /// Transaction group.
-    pub TimeTransactions {
+/// Timestamping transaction.
+#[derive(Serialize, Deserialize, Clone, Debug, ProtobufConvert)]
+#[protobuf_convert("proto::TxTimestamp")]
```

Comment:
Nit: I wonder if it's possible to come up with a reasonable default convention to skip specifying this attribute in most cases. Say, place Protobuf files in the `proto` module of the crate, so we can include them via `crate::proto::#name`.